src/AdminConsole/FeatureManagement/OrganizationFeatureFilter.cs

```diff
@@ -0,0 +1,36 @@
+using Microsoft.FeatureManagement;
+using Passwordless.AdminConsole.Authorization;
+
+namespace Passwordless.AdminConsole.FeatureManagement;
+
+[FilterAlias("Organization")]
+public class OrganizationFeatureFilter : IFeatureFilter
+{
+    private readonly IHttpContextAccessor _httpContextAccessor;
+
+    public OrganizationFeatureFilter(IHttpContextAccessor httpContextAccessor)
+    {
+        _httpContextAccessor = httpContextAccessor;
+    }
+
+    public Task<bool> EvaluateAsync(FeatureFilterEvaluationContext context)
+    {
+        var httpContext = _httpContextAccessor.HttpContext!;
+
+        if (httpContext.User.Identity is { IsAuthenticated: true })
+        {
+            var organizationIdClaim = httpContext.User.FindFirst(CustomClaimTypes.OrgId);
```

Comment:
Could you use `.GetOrgId()` extension instead?

Reply:
@jrmccannon Check again. I also made it return `int` instead of it's nullable representation. Given a logged in admin always has an organization id or `OrgId` claim.